packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -952,6 +952,36 @@ describe('TransactionController', () => {
       );
     });
 
+    it('adds initial history', async () => {
+      const controller = newController();
+
+      await controller.addTransaction({
+        from: ACCOUNT_MOCK,
+        to: ACCOUNT_MOCK,
+      });
+
+      expect(controller.state.transactions[0]?.history).toBeDefined();
```

Comment:
Should we be explicit as possible with the assertions to ensure the state itself has the exact structure we expect based on the mock data we added?